sdk/devcenter/developer-devcenter-rest/samples/v1/javascript/sampleCreateDevBox.js

```diff
@@ -0,0 +1,92 @@
+const { DefaultAzureCredential } = require("@azure/identity");
+const { isUnexpected, getLongRunningPoller } = require("@azure-rest/developer-devcenter");
+const createClient = require("@azure-rest/developer-devcenter").default;
+
+/**
+ * @summary Demonstrates creating, accessing, and deleting a Dev Box
+ */
+async function createDevBox() {
+  // Build client and fetch required parameters
+  const tenantId = process.env.AZURE_TENANT_ID || "<tenant id>";
+  const devCenter = process.env.AZURE_DEVCENTER_NAME || "<devcenter name>";
+  const client = createClient(tenantId, devCenter, new DefaultAzureCredential());
+
+  const projectList = await client.path("/projects").get();
+  if (isUnexpected(projectList)) {
+    throw projectList.body.error;
+  }
+
+  let project = projectList.body.value[0];
+  if (project === undefined || project.name === undefined) {
```

Comment:
Could you reflect your sample changes to samples folder by running below command?

```
cd ${PROJECT_ROOT}
dev-tool samples publish -f
```